docs/guide/v2_upgrade_guide.md

```diff
@@ -246,3 +246,47 @@ val entity = dyDatabase.dyTable.dyEntitiy.load(
 + consistentReads = true
 )
 ```
+
+## Changes to Misk
+
+If you are using [Misk](https://github.com/cashapp/misk) you will need to update your dependencies and DI configuration
+
+### Dependencies
+
+```diff
+-val miskAwsDynamodb = "com.squareup.misk:misk-aws-dynamodb:VERSION"
+-val miskAwsDynamodbTesting = "com.squareup.misk:misk-aws-dynamodb-testing:VERSION"
++val miskAws2Dynamodb = "com.squareup.misk:misk-aws2-dynamodb:VERSION"
++val miskAws2DynamodbTesting = "com.squareup.misk:misk-aws2-dynamodb-testing:VERSION"
+```
+
+### DynamoDbModule
```

Comment:
I think all the significant changes here are really in the AWS `ClientOverrideConfiguration`...so this is less of a misk thing and more of an AWS SDK upgrade concern?

Reply:
Glad you caught this, I copied this from the wrong level. I updated it to show the Misk module configuration.